src/ripple/protocol/Feature.h

```diff
@@ -360,6 +360,7 @@ extern uint256 const fixEmptyDID;
 extern uint256 const fixXChainRewardRounding;
 extern uint256 const fixPreviousTxnID;
 extern uint256 const fixAMMRounding;
+extern uint256 const fixAMMMetadata;
```

Comment:
Having read https://github.com/XRPLF/rippled/issues/4965#issuecomment-2079688630 I think this is a good fix, but I would like the amendment to use a different name. The use of `AMM` in name is misleading, this is a currency metadata fix (i.e. a lower abstraction level), which just happens to show up in AMM (but also in Oracle, as the unit tests demonstrate).

Reply:
Could be also named as asset metadata, rather than currency metadata.